src/styles/palette.less

```diff
@@ -15,30 +15,30 @@
 
 @import "branding";//branding.less
 
-@inactivecolor: #DCDDDE;
-@subheadingcolor: #5F5F5F;
-@defaultcolor: #5F5F5F;
+@inactivecolor: @sb-gray-100;
+@subheadingcolor: @sb-gray-900;
+@defaultcolor: @sb-gray-900;
 
 @errorcolor: #ff420a;
 @error-icon-color: #a94442;
 
-@defaultshadow: #c5c5c5;
+@defaultshadow: @sb-gray-200;
 
-@listbodycolor: #EDEDED;
+@listbodycolor: @sb-gray-100;
 @dropdowncolor: #5F5F5F;
 @detailcolor: #58595B;
 
-@page-background-color: #f7f7f7;
+@page-background-color: @sb-white;
 @page-font-color: #3a3e40;
 @table-header-gradient-color: #f4f4f4;
 @table-header-color: #e7e7e7;
 @box-border-color: #cccccc;
-@info-link-color: #337ab7;
+@info-link-color: @sb-cerulean-800;
```

Comment:
Should use `@sb-cerulean-900`

Reply:
This is the same link color used on the eos design app